[BUG] Issue with ScriptAPI's Execution Context Environment

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.1.5 git-02de191

### Operating System

Windows 10

### Minecraft Version

1.20.4

### Describe the bug

An exception is thrown when attempting to use Java objects in the script: "ReferenceError: "Java" is not defined."

### Steps to reproduce

Create a JavaScript like this in the scripts directory:
```javascript
// Test.js
const LiquidBounce = Java.type("net.ccbluex.liquidbounce.LiquidBounce");

const script = registerScript({
    name: "Test",
    version: "0.1",
    authors: ["mumy++"]
});

script.on("load", () => {
    LiquidBounce.INSTANCE.getLogger().info("[Test] load!");
});

script.on("enable", () => {
    LiquidBounce.INSTANCE.getLogger().info("[Test] enable!");
});

script.on("disable", () => {
    LiquidBounce.INSTANCE.getLogger().info("[Test] disable!");
});
```

### Client Log

```shell
[19:10:33] [main/INFO]: Loading Minecraft 1.20.4 with Fabric Loader 0.15.6
[19:10:33] [ForkJoinPool-1-worker-3/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR1-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[19:10:34] [main/INFO]: Loading 119 mods:
	- customskinloader 14.19.1
	- disable_custom_worlds_advice 5.0
	- fabric-api 0.96.4+1.20.4
	   |-- fabric-api-base 0.4.36+78d798af4f
	   |-- fabric-api-lookup-api-v1 1.6.49+82b1bb3e4f
	   |-- fabric-biome-api-v1 13.0.16+78d798af4f
	   |-- fabric-block-api-v1 1.0.16+3e2216cb4f
	   |-- fabric-block-view-api-v2 1.0.4+78d798af4f
	   |-- fabric-blockrenderlayer-v1 1.1.46+78d798af4f
	   |-- fabric-client-tags-api-v1 1.1.7+78d798af4f
	   |-- fabric-command-api-v1 1.2.41+f71b366f4f
	   |-- fabric-command-api-v2 2.2.20+78d798af4f
	   |-- fabric-commands-v0 0.2.58+df3654b34f
	   |-- fabric-containers-v0 0.1.86+df3654b34f
	   |-- fabric-content-registries-v0 5.0.14+78d798af4f
	   |-- fabric-convention-tags-v1 1.5.10+78d798af4f
	   |-- fabric-crash-report-info-v1 0.2.23+78d798af4f
	   |-- fabric-data-attachment-api-v1 1.1.3+b90db5744f
	   |-- fabric-data-generation-api-v1 13.2.3+5c0133444f
	   |-- fabric-dimensions-v1 2.1.61+78d798af4f
	   |-- fabric-entity-events-v1 1.6.1+09fc25014f
	   |-- fabric-events-interaction-v0 0.7.1+389931eb4f
	   |-- fabric-events-lifecycle-v0 0.2.74+df3654b34f
	   |-- fabric-game-rule-api-v1 1.0.46+78d798af4f
	   |-- fabric-item-api-v1 2.2.0+d6f2b0844f
	   |-- fabric-item-group-api-v1 4.0.25+58f8c0124f
	   |-- fabric-key-binding-api-v1 1.0.41+78d798af4f
	   |-- fabric-keybindings-v0 0.2.39+df3654b34f
	   |-- fabric-lifecycle-events-v1 2.3.0+a67ffb5d4f
	   |-- fabric-loot-api-v2 2.1.8+78d798af4f
	   |-- fabric-message-api-v1 6.0.5+78d798af4f
	   |-- fabric-mining-level-api-v1 2.1.64+78d798af4f
	   |-- fabric-model-loading-api-v1 1.0.8+78d798af4f
	   |-- fabric-models-v0 0.4.7+9386d8a74f
	   |-- fabric-networking-api-v1 3.1.7+2e5ac5484f
	   |-- fabric-object-builder-api-v1 13.0.13+080016e44f
	   |-- fabric-particles-v1 1.1.7+78d798af4f
	   |-- fabric-recipe-api-v1 2.0.20+78d798af4f
	   |-- fabric-registry-sync-v0 4.0.19+58f8c0124f
	   |-- fabric-renderer-api-v1 3.2.4+78d798af4f
	   |-- fabric-renderer-indigo 1.5.4+78d798af4f
	   |-- fabric-renderer-registries-v1 3.2.53+df3654b34f
	   |-- fabric-rendering-data-attachment-v1 0.3.42+73761d2e4f
	   |-- fabric-rendering-fluids-v1 3.1.1+e761c6694f
	   |-- fabric-rendering-v0 1.1.56+df3654b34f
	   |-- fabric-rendering-v1 3.2.0+6fd945a04f
	   |-- fabric-resource-conditions-api-v1 2.3.14+78d798af4f
	   |-- fabric-resource-loader-v0 0.11.19+58f8c0124f
	   |-- fabric-screen-api-v1 2.0.17+78d798af4f
	   |-- fabric-screen-handler-api-v1 1.3.55+78d798af4f
	   |-- fabric-sound-api-v1 1.0.17+78d798af4f
	   |-- fabric-transfer-api-v1 4.0.11+e84342304f
	   \-- fabric-transitive-access-wideners-v1 5.0.14+78d798af4f
	- fabric-language-kotlin 1.10.18+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.6
	   \-- mixinextras 0.3.2
	- indium 1.0.30+mc1.20.4
	- java 17
	- krypton 0.2.6
	   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
	- lambdabettergrass 1.5.2+1.20.1
	- lambdynlights 2.3.4+1.20.4
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 5.0.3+1.20.2
	- lazydfu 0.1.3
	- liquidbounce 0.1.5
	- lithium 0.12.1
	- logical_zoom 0.0.22
	- memoryleakfix 1.1.5
	- minecraft 1.20.4
	- modernfix 5.13.0+mc1.20.4
	- modmenu 9.0.0
	- nochatreports 1.20.4-v2.5.0
	- packetfixer 1.2.4
	- reeses-sodium-options 1.7.2+mc1.20.4-build.102
	- shulkerboxtooltip 4.0.8+1.20.4
	   \-- cloth-config 13.0.121
	        \-- cloth-basic-math 0.6.1
	- sodium 0.5.8+mc1.20.4
	- sodium-extra 0.5.4+mc1.20.4-build.116
	   |-- caffeineconfig 1.3.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
	- starlight 1.1.3+fabric.f5dcd1a
	- viafabricplus 3.0.6
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viabackwards-common 4.10.0-23w51b-SNAPSHOT
	   |-- com_viaversion_viaversion-common 4.10.0-23w51b-SNAPSHOT
	   |-- de_florianmichael_classic4j 2.0.1
	   |-- io_jsonwebtoken_jjwt-api 0.12.3
	   |-- io_jsonwebtoken_jjwt-gson 0.12.3
	   |-- io_jsonwebtoken_jjwt-impl 0.12.3
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_mcping 1.4.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.0
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.0
	   |-- net_lenni0451_reflect 1.3.1
	   |-- net_raphimc_minecraftauth 3.1.0
	   |-- net_raphimc_viaaprilfools 2.0.11-SNAPSHOT
	   |-- net_raphimc_viabedrock 0.0.4-SNAPSHOT
	   |-- net_raphimc_vialegacy 2.2.22-SNAPSHOT
	   |-- net_raphimc_vialoader 2.2.13-SNAPSHOT
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR1-SNAPSHOT
	   |-- org_iq80_leveldb_leveldb 0.12
	   |-- org_iq80_leveldb_leveldb-api 0.12
	   |-- org_lz4_lz4-pure-java 1.8.0
	   \-- org_yaml_snakeyaml 2.2
	- xaerominimap 23.9.7
[19:10:34] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/H:/Games/Play2/.minecraft/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[19:10:34] [main/INFO]: Compatibility level set to JAVA_17
[19:10:35] [main/INFO]: Loaded configuration file for Lithium: 129 options available, 0 override(s) found
[19:10:35] [main/WARN]: Reference map 'memoryleakfix-fabric-1.17+-fabric-refmap.json' for memoryleakfix-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[19:10:35] [main/INFO]: Loaded configuration file for ModernFix 5.13.0+mc1.20.4: 52 options available, 0 override(s) found
[19:10:35] [main/INFO]: Applying Nashorn fix
[19:10:35] [main/WARN]: Reference map 'PacketFixer-fabric-fabric-refmap.json' for packetfixer-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[19:10:35] [main/INFO]: Loaded configuration file for Sodium: 42 options available, 0 override(s) found
[19:10:35] [main/INFO]: Loaded configuration file for Sodium Extra: 34 options available, 0 override(s) found
[19:10:36] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21FrameDecoderMixin). Delete Krypton or Pluto.
[19:10:36] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21LengthFieldPrependerMixin). Delete Krypton or Pluto.
[19:10:36] [main/INFO]: [MemoryLeakFix] Will be applying 3 memory leak fixes!
[19:10:36] [main/INFO]: [MemoryLeakFix] Currently enabled memory leak fixes: [targetEntityLeak, biomeTemperatureLeak, hugeScreenshotLeak]
[19:10:36] [main/INFO]: Searching for graphics cards...
[19:10:36] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.2).
[19:10:37] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=NVIDIA, name=NVIDIA GeForce GTX 1050 Ti, version=DriverVersion=31.0.15.3623]
[19:10:37] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=UNKNOWN, name=RayLink Virtual Display Adapter, version=DriverVersion=1.0.2.0]
[19:10:37] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS]
[19:10:37] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[19:10:39] [main/INFO]: liquidbounce.mixins.json:minecraft.entity.MixinPlayerEntityAccessor from mod liquidbounce->@Accessor[FIELD_GETTER]::getTrackedMainArm()Lnet/minecraft/class_2940; should be static as its target is
[19:10:40] [main/INFO]: Injecting BlockStateBase cache population hook into getNeighborPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[19:10:40] [main/INFO]: Injecting BlockStateBase cache population hook into getPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[19:10:40] [main/INFO]: Injecting BlockStateBase cache population hook into getOpacityIfCached from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[19:10:40] [main/INFO]: Injecting BlockStateBase cache population hook into isConditionallyFullOpaque from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[19:10:40] [main/INFO]: Injecting BlockStateBase cache population hook into lithium$getAllFlags from me.jellysquid.mods.lithium.mixin.util.block_tracking.AbstractBlockStateMixin
[19:10:41] [main/WARN]: Method overwrite conflict for method_21740 in modernfix-common.mixins.json:perf.remove_biome_temperature_cache.BiomeMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.temperature_cache.BiomeMixin. Skipping method.
[19:10:44] [main/INFO]: Vanilla bootstrap took 5974 milliseconds
[19:10:45] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:10:45] [Render thread/INFO]: Setting user: ____
[19:10:45] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[19:10:45] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[19:10:45] [Render thread/INFO]: Bypassed Mojang DFU
[19:10:45] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[19:10:45] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[19:10:46] [Render thread/INFO]: Compression will use Java, encryption will use Java
[19:10:46] [Render thread/INFO]: Packet Fixer has been initialized successfully
[19:10:46] [Download-2/ERROR]: Failed to fetch user properties
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
	at com.mojang.authlib.exceptions.MinecraftClientHttpException.toAuthenticationException(MinecraftClientHttpException.java:59) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:150) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_310.method_55608(class_310.java:467) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.get(MinecraftClient.java:57) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:121) ~[authlib-6.0.52.jar:?]
	... 5 more
[19:10:46] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[19:10:46] [Render thread/INFO]: Krypton is now accelerating your Minecraft client's networking stack ?
[19:10:46] [Render thread/INFO]: Note that Krypton is most effective on servers, not the client.
[19:10:46] [Render thread/INFO]: [LambdaBetterGrass] Initializing LambdaBetterGrass...
[19:10:46] [Render thread/INFO]: [LambDynLights] Initializing LambDynamicLights...
[19:10:46] [Render thread/INFO]: [LambDynLights] Configuration loaded.
[19:10:47] [Worker-Main-1/INFO]: Checking mod updates...
[19:10:47] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:8178/
[19:10:47] [Render thread/INFO]: Loading Xaero's Minimap - Stage 1/2
[19:10:47] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: 1.7.2-1.7.5 (4)
[19:10:47] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 23w51b (766)
[19:10:47] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:10:47] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[19:10:48] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[19:10:48] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[19:10:48] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
[19:10:48] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 536.23
[19:10:48] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[19:10:49] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[19:10:49] [Render thread/INFO]: Loaded language en_us
[19:10:49] [Render thread/INFO]: Loaded language de_de
[19:10:49] [Render thread/INFO]: Loaded language ja_jp
[19:10:49] [Render thread/INFO]: Loaded language zh_cn
[19:10:49] [Render thread/INFO]: Loaded language ru_ru
[19:10:49] [Render thread/INFO]: Loaded language en_pt
[19:10:49] [Render thread/INFO]: Loaded language pt_br
[19:10:52] [Finalizer/WARN]: One or more BufferBuilders have been leaked, ModernFix will attempt to correct this.
[19:10:52] [Render thread/ERROR]: Unable to load script Test.js.
org.graalvm.polyglot.PolyglotException: ReferenceError: "Java" is not defined
	at <js>.:program(Test.js:2) ~[?:?]
	at org.graalvm.polyglot.Context.eval(Context.java:403) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.script.Script.initScript(Script.kt:89) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.script.ScriptManager.loadScript(ScriptManager.kt:87) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.script.ScriptManager.loadSafely(ScriptManager.kt:77) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.script.ScriptManager.loadScripts(ScriptManager.kt:60) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:154) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:174) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dai000$liquidbounce$startClient(class_310.java:15847) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.6.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.6.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[19:10:53] [Render thread/INFO]: Successfully extracted default theme
[19:10:53] [Render thread/INFO]: Inserted 5 components
[19:10:53] [Render thread/INFO]: Successfully loaded config 'enemies'.
[19:10:53] [Render thread/INFO]: (Chat) 
[19:10:53] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7Auto ConfigAuto Config
[19:10:53] [Render thread/INFO]: Successfully loaded config 'modules'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'commands'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'friends'.
[19:10:53] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:275) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:230) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:205) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:146) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:174) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dai000$liquidbounce$startClient(class_310.java:15847) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.6.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.6.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 24 more
[19:10:53] [Render thread/INFO]: Successfully loaded config 'proxy'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'accounts'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'tabs'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'lang'.
[19:10:53] [Render thread/INFO]: Successfully loaded config 'storage'.
[19:10:53] [netty-websocket/INFO]: Starting Netty server...
[19:10:53] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d] REGISTERED
[19:10:53] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d] BIND: 0.0.0.0/0.0.0.0:15000
[19:10:53] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] ACTIVE
[19:10:53] [netty-websocket/INFO]: Netty server started on port 15000.
[19:10:54] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[19:10:54] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[19:10:54] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[19:10:54] [Render thread/INFO]: https://dl.liquidbounce.net/resources/java-cef-builds/e91a133673f9524c5f7db9bdbad1aff61b3887d7/windows_amd64.tar.gz.sha256 -> H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.6\LiquidBounce\mcef\libraries\windows_amd64.tar.gz.sha256.temp
[19:10:57] [Render thread/INFO]: Checksum matches: true
[19:10:57] [Render thread/INFO]: Platform directory exists: true
[19:10:57] [Render thread/INFO]: Initializing CEF on windows_amd64...
[19:10:57] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.6\LiquidBounce\mcef\libraries\windows_amd64
[19:10:57] [Render thread/INFO]: Chromium Embedded Framework initialized
[19:10:57] [Render thread/INFO]: Browser is ready.
[19:10:57] [Render thread/INFO]: Successfully loaded client!
[19:10:58] [Render thread/INFO]: Invalidating pack caches
[19:10:58] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, caffeineconfig, cloth-config, com_viaversion_viabackwards-common, com_viaversion_viaversion-common, disable_custom_worlds_advice, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, indium, krypton, lambdabettergrass, lambdynlights, lazydfu, liquidbounce, lithium, logical_zoom, modernfix, modmenu, net_raphimc_viaaprilfools, net_raphimc_viabedrock, net_raphimc_vialegacy, nochatreports, packetfixer, pride, reeses-sodium-options, shulkerboxtooltip, sodium, sodium-extra, spruceui, starlight, viafabricplus, xaerominimap
[19:10:58] [Render thread/INFO]: [LambdaBetterGrass] Inject generated resource pack.
[19:10:58] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[19:10:58] [Worker-Main-1/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$ced000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[19:10:58] [Worker-Main-1/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$ced000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[19:10:58] [Render thread/INFO]: Created item group Heads
[19:10:58] [Render thread/INFO]: Created item group Exploits
[19:10:58] [Render thread/INFO]: Created item group Containers
[19:10:58] [Render thread/INFO]: Successfully setup tabs
[19:10:58] [Render thread/INFO]: Loading Xaero's Minimap - Stage 2/2
[19:10:58] [Worker-ResourceReload-2/INFO]: Found unifont_all_no_pua-15.1.04.hex, loading
[19:11:02] [Render thread/INFO]: Registered player tracker system: minimap_synced
[19:11:02] [Render thread/INFO]: No Optifine!
[19:11:02] [Render thread/INFO]: Xaero's Minimap: No Vivecraft!
[19:11:03] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[19:11:03] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[19:11:03] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on 扬声器 (Realtek High Definition Audio)
[19:11:03] [Render thread/INFO]: Sound engine started
[19:11:03] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[19:11:03] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[19:11:03] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[19:11:03] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[19:11:03] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[19:11:03] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[19:11:03] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[19:11:03] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[19:11:03] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[19:11:04] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[19:11:04] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[19:11:04] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[19:11:04] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[19:11:04] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[19:11:04] [Render thread/INFO]: Loading fonts...
[19:11:04] [Render thread/INFO]: Loading queued font Montserrat
[19:11:04] [Render thread/INFO]: Loaded fonts successfully!
[19:11:06] [Render thread/INFO]: Loading Discord IPC configuration...
[19:11:06] [Render thread/INFO]: Loaded Discord IPC configuration.
[19:11:06] [Render thread/INFO]: Refreshing local IP info...
[19:11:07] [Render thread/INFO]: Loading heads...
[19:11:07] [nioEventLoopGroup-5-1/INFO]: IP Info [CN, AS17816 China Unicom IP network China169 Guangdong province]
[19:11:10] [Render thread/INFO]: Successfully loaded 956 heads from the database
[19:11:10] [Render thread/INFO]: Loading settings list from API...
[19:11:11] [Render thread/INFO]: Loaded 25 settings from API.
[19:11:11] [Render thread/INFO]: Successfully loaded 233 cape carriers.
[19:11:11] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[19:11:11] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@6a4e5a80, url=http://127.0.0.1:15000/default/#/)
[19:11:11] [Render thread/INFO]: Successfully reloaded the minimap shaders!
[19:11:11] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xea5ac4f0, L:/127.0.0.1:15000 - R:/127.0.0.1:8219]
[19:11:11] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:12] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[19:11:12] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@6a4e5a80, url=http://127.0.0.1:15000/default/#/)
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x2fc3c31f, L:/127.0.0.1:15000 - R:/127.0.0.1:8233]
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xeb726b0d, L:/127.0.0.1:15000 - R:/127.0.0.1:8235]
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xa4bfbd75, L:/127.0.0.1:15000 - R:/127.0.0.1:8238]
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x8064d618, L:/127.0.0.1:15000 - R:/127.0.0.1:8239]
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x202876ee, L:/127.0.0.1:15000 - R:/127.0.0.1:8240]
[19:11:12] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:13] [Render thread/WARN]: Game took 41.618 seconds to start
[19:11:13] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[19:11:13] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@6a4e5a80, url=http://127.0.0.1:15000/default/#/)
[19:11:14] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[19:11:14] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@6a4e5a80, url=http://127.0.0.1:15000/default/#/)
[19:11:14] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x59cd2787, L:/127.0.0.1:15000 - R:/127.0.0.1:8253]
[19:11:14] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x33c0e4ab, L:/127.0.0.1:15000 - R:/127.0.0.1:8255]
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:15] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7UI desync detected, it should now be fixed!UI desync detected, it should now be fixed!
[19:11:15] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@6a4e5a80, url=http://127.0.0.1:15000/default/#/)
[19:11:15] [nioEventLoopGroup-4-6/WARN]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
java.io.IOException: 你的主机中的软件中止了一个已建立的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:46) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:330) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:284) ~[?:?]
	at sun.nio.ch.IOUtil.read(IOUtil.java:259) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:417) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:254) ~[netty-buffer-4.1.97.Final.jar:?]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.97.Final.jar:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.97.Final.jar:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xa7b20a33, L:/127.0.0.1:15000 - R:/127.0.0.1:8258]
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0x658430d0, L:/127.0.0.1:15000 - R:/127.0.0.1:8259]
[19:11:15] [nioEventLoopGroup-3-1/INFO]: [id: 0xd3ce2e5d, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[19:12:08] [Render thread/ERROR]: ########## GL ERROR ##########
[19:12:08] [Render thread/ERROR]: @ Render
[19:12:08] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushAttrib is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glOrtho is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glMatrixMode is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glPushMatrix is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Function glLoadIdentity is deprecated and not available in preview contexts.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'
[19:12:10] [Thread-601/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_ENUM error generated. Cannot enable <cap> in the current profile.'

(The following are similar log information being repeated)
```


### Screenshots

_No response_

--- a/src/main/kotlin/net/ccbluex/liquidbounce/script/Script.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/script/Script.kt
@@ -41,6 +41,7 @@ class Script(val scriptFile: File) {
 
     private val context: Context = Context.newBuilder("js")
         .allowHostAccess(HostAccess.ALL) // Allow access to all Java classes
+        .allowHostClassLookup { true }
         .currentWorkingDirectory(scriptFile.parentFile.toPath())
         .allowIO(IOAccess.ALL) // Allow access to all IO operations
         .allowCreateProcess(false) // Disable process creation

